DOCUMENTATION.md

```diff
@@ -648,6 +648,23 @@ id = GetId(e);
 
 returns value `ns0:bob`
 
+#### SetId
+
+`SetId(entity, id)` takes a parameter of type `Entity` and a string with an id, and updates the id of the entity.
+
+Example:
+
+```javascript
+var e = NewEntity();
+SetId(e, PrefixField("ns0", 42));
```

Comment:
while this does the right thing its a bit confusing perhaps from a naming perspective, as we are passing a value not a field, it just happens these are both CURIEs.